### PR TITLE
BACKLOG-12244 : use callback to register actions

### PR DESCRIPTION
**Pull request by dgriffon**
https://jira.jahia.org/browse/BACKLOG-12244

### CODE REVIEW

File: src/javascript/Create/index.js [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-
-    // In app actions
-    registry.add('action', 'createButton', createButtonAction, {
-        buttonLabel:
-            'content-editor:label.contentEditor.create.createButton.name',
-        buttonIcon: <Save/>,
-        targets: ['editHeaderActions:1']
-    });
-};
+export default registerActions;
```
Comment (tdraier) [inline src/javascript/Create/index.js]: use a named export here

File: src/javascript/Edit/index.js [hunk 2]
```diff
@@ -1,94 +1,3 @@
...
-        });
-
-        actionsRegistry.add('action', 'usagesTabAction', openEngineTabs, {
-            buttonLabel: 'content-editor:label.contentEditor.edit.action.usagesTab',
-            targets: ['ContentEditorHeaderActions:7'],
-            tabs: ['usages']
-        });
-    }
-};
+export default registerActions;
```
Comment (tdraier) [inline src/javascript/Edit/index.js]: use a named export here

File: src/javascript/registerCEActions.jsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import {ArrowBack} from '@material-ui/icons';
 import {SelectorTypes} from '~/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/SelectorTypes';
-import {registerActions as registerEditActions} from '~/Edit';
-import {registerActions as registerCreateActions} from '~/Create';
+import registerEditActions from '~/Edit';
+import registerCreateActions from '~/Create';
```
Comment (tdraier) [inline src/javascript/registerCEActions.jsx]: import from ~/Edit/registerEditActions  and ~/Create/registerCreateActions ( or eventually rename to [Edit|Create].actions.js to be consistent with cmm )

File: src/javascript/Create/index.js (hunk 1 above)
Comment (dgriffon) [reply src/javascript/Create/index.js]: file has been removed

File: src/javascript/Edit/index.js (hunk 2 above)
Comment (dgriffon) [reply src/javascript/Edit/index.js]: file has been removed

File: src/javascript/registerCEActions.jsx (hunk 3 above)
Comment (dgriffon) [reply src/javascript/registerCEActions.jsx]: named exports and same names as CMM is now used here

**Pull request merged**